<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextNodeDemoClass</class>
 <widget class="QMainWindow" name="TextNodeDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextNodeDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>891</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_NEW_N"/>
    <addaction name="action_O"/>
    <addaction name="action_S"/>
    <addaction name="action_A"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="action_T"/>
    <addaction name="action_C"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="action_W"/>
    <addaction name="action_F"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="action_S_2"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_A_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_NEW_N">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>打开（O）</string>
   </property>
   <property name="toolTip">
    <string>打开（O）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>保存（S）</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>另存为（A）</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>退出（X）</string>
   </property>
  </action>
  <action name="action_N">
   <property name="text">
    <string>撤销（N）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_T">
   <property name="text">
    <string>剪切（T）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>复制（C）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_W">
   <property name="text">
    <string>自动换行（W）</string>
   </property>
  </action>
  <action name="action_F">
   <property name="text">
    <string>字体（F）</string>
   </property>
  </action>
  <action name="action_S_2">
   <property name="text">
    <string>状态栏（S）</string>
   </property>
  </action>
  <action name="action_A_2">
   <property name="text">
    <string>关于记事本（A）</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="textnodedemo.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>PushButtonClicked()</slot>
 </slots>
</ui>
